exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/SimpleRepeatedFunctions.java

```diff
@@ -77,8 +79,6 @@ public void eval() {
     }
   }
 
-  /*
-  // TODO - need to confirm that these work   SMP: They do not
```

Comment:
What about TODO?
> // TODO - replace with a freemarker template and fill out the rest of the types
>   // focused on getting functions defined for JSON types as this is the primary format
>   // users are extracting repeated data out of currently

May it be used for other datatypes?

Reply:
Not sure what you mean. Do you mean to create a template instead as the TODO suggests?

Reply:
> Not sure what you mean. Do you mean to create a template instead as the TODO suggests?

Yes. But I am not sure that it will save much more space for current `repeated_count` functions, since there are different implementations of `eval()` method. But if some additional data types can be used with this function, then it makes sense to add them and to consider rewriting this calss with FreeMarker template.
So can other data types be used with `repeated_count` function?

Reply:
Moved `repeated_count` functions with uniform implementation to a separate freemarker template file. Thanks for pointing the `TODO` out :+1: